packages/grid/backend/backend.dockerfile

```diff
@@ -35,7 +35,7 @@ RUN --mount=type=cache,target=/root/.cache,sharing=locked \
     # remove torch because we already have the cpu version pre-installed
     sed --in-place /torch==/d ./syft/setup.cfg && \
     uv pip install -e ./syft[data_science] && \
-    uv pip freeze | grep ansible | xargs uv pip uninstall
+    if uv pip freeze | grep -q ansible; then uv pip freeze | grep ansible | xargs uv pip uninstall; fi
```

Comment:
Since hagrid is removed as a dependency , we could actually remove this statement, as ansible was previously installed by hagrid.

We could keep this to new PR.
Merging the PR now